_docs/README.md

```diff
@@ -0,0 +1,275 @@
+# Boilerplate
+
+Boilerplate is a tool for generating files and folders ("boilerplate") from a set of templates.
+
+Example use cases:
+
+1. Create the scaffolding for a new project (e.g. like [html5boilerplate](https://html5boilerplate.com/))
+1. Fill in boilerplate sections in your code files, such as including a legal disclaimer or license at the top of each
+   source file, or updating a version number in a file after each build.
+1. Embed code snippets from actual source files in documentation. Most READMEs have code copy/pasted into them and that
+   code often has syntax errors or goes out of date. Now you can keep those code examples in normal source files which
+   are built & tested, and embed parts of those files dynamically in your docs.
+
+Note: the README for this project is generated using boilerplate! Check out the templates for it in the
+[_docs folder](/_docs) and the build job configuration in [circle.yml](/circle.yml).
+
+## Example: creating a new template
+
+Create a folder called `website-boilerplate` and put a file called `boilerplate.yml` in it:
+
+```json
+{{snippet "../examples/website/boilerplate.yml"}}
+```
```

Comment:
Oh, interesting, I just realized that `{{snippet ...}}` will be replaced by boilerplate, so I guess you want "```yml" here.